<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManageForm</class>
 <widget class="QWidget" name="UserManageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="userinfoLabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>当前选择的用户是：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addUserBtn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="alterUserinfoBtn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改用户信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeUserBtn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>移除用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bookRecordsBtn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查看借书记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>多行删除</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(122, 122, 122);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <column>
     <property name="text">
      <string>Id</string>
     </property>
     <property name="background">
      <color>
       <red>111</red>
       <green>111</green>
       <blue>111</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>用户名</string>
     </property>
     <property name="background">
      <color>
       <red>111</red>
       <green>111</green>
       <blue>111</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>未还书籍数</string>
     </property>
     <property name="background">
      <color>
       <red>111</red>
       <green>111</green>
       <blue>111</blue>
      </color>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="renovateBtn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>260</y>
     <width>301</width>
     <height>123</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="IdEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="IdQueryBtn">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Id查询</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="nameEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="nameQueryBtn">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>用户名查询</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
